modules/stanford_course_views/stanford_course_views.views_default.inc

```diff
@@ -185,8 +185,8 @@ function stanford_course_views_views_default_views() {
   $handler->display->display_options['empty']['area']['empty'] = TRUE;
   $handler->display->display_options['empty']['area']['content'] = '<p>Current Courses displays all courses offered in the current quarter.</p>
 <p>&nbsp;</p>
-[hook_views_pre_build:courses/search]';
-  $handler->display->display_options['empty']['area']['format'] = 'filtered_html';
+<a class="btn" href="/courses/search">Search courses</a>';
+  $handler->display->display_options['empty']['area']['format'] = 'content_editor_text_format';
```

Comment:
Yes, we have made many mistakes in the past. If we wanted to be fully KIT compliant with all of our features we would need a pretty massive overhaul. If we ever get to a point where we need to do that it will be nice to have one less thing to have to fix.